### PR TITLE
docs: Add contributing guidelines

### PR DESCRIPTION
**Pull request by Mindavi**
Let's discuss the format we want to use for contributions/commits.

Is it okay to move the 'adding new devices' section from README to CONTRIBUTING?

### CODE REVIEW

**Comment by zuckschwerdt** [comment]
Great! I'll have a closer look tomorrow. Perhaps best to place the file in `docs` for easier building of the documentation site and to unclutter the root.

**Comment by zuckschwerdt** [comment]
Sorry, I forgot to "Submit review" so my notes were not visible.
@merbanan thoughts on "prefix everything" vs. "prefix only the housekeeping" (esp. not the very common decoder changes)?

**Comment by merbanan** [comment]
@zuckschwerdt, prefix house keeping like cosmetics and such

**Comment by zuckschwerdt** [comment]
@Mindavi can you remove the "<decoder_name>:", "decoders:", "core:" mentions and have
```
The commit messages should follow the common format of

<area_of_work>: <verb> <commit_message>

Area of work is optional and may be one of the following: 
```

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+- build: for build/build system related work
+- docs: for documentation related work, both in code and readme/docs folder
+- ci: for work related to continuous integration
+- test: for test related work 
+- <decoder_name>: for decoder related work
+- decoders: when multiple decoders are changed at once
+- deps: for changes related to (external) dependencies (e.g. soapysdr is updated or mongoose is updated)
+- core: for changes related to the core decoding and parsing logic (e.g. bitbuffer parsing, fsk decoding)
+- minor: for housekeeping work, code style changes
```
Comment (merbanan) [inline docs/CONTRIBUTING.md]: I use cosmetics for things that are non functional changes. Indentation for example.

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- Change: for changes that modify input/output behaviour (e.g. added checksums, preambles)
+- Improve: for improvements without changes in normal output/behaviour
+
+## Supporting Additional Devices and Test Data
+
+Some device protocol decoders are disabled by default. When testing to see if your device
+is decoded by rtl_433, use `-G` to enable all device protocols.
+This will likely produce false positives, use with caution.
+
+The first step in decoding new devices is to record the signals using `-S all`.
```
Comment (merbanan) [inline docs/CONTRIBUTING.md]: -S unknown?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Contributing guidelines
+
+The rtl_433 project accepts contributions.
```
[the remark this answers is not in the document]
Comment (Mindavi) [reply CONTRIBUTING.md]: That's way better indeed.

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (Mindavi) [reply docs/CONTRIBUTING.md]: Good catch

Comment (Mindavi) [reply docs/CONTRIBUTING.md]: I think the section might need some more love, but that's out of scope for this PR. I'm just moving it from the readme.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+<area_of_work>: <verb> <commit_message>
+
+Area of work may be one of the following:
+
+- build: for build/build system related work
+- docs: for documentation related work, both in code and readme/docs folder
+- ci: for work related to continuous integration
+- test: for test related work 
+- <decoder_name>: for decoder related work
```
[the remark this answers is not in the document]
Comment (Mindavi) [reply CONTRIBUTING.md]: I'm okay with that.

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (Mindavi) [reply docs/CONTRIBUTING.md]: Let's leave it at cosmetics then.

**Comment by Mindavi** [comment]
@zuckschwerdt is adding the documentation to the doc website as simple as putting it in the `docs/.vuepress/config.js` file? Is the format correct for building correctly?

**Comment by zuckschwerdt** [comment]
Yes, that's all. I just need to build and upload manually for now.

**Pull request merged**